Userland/Applications/SystemMonitor/NetworkStatisticsWidget.cpp

```diff
@@ -26,6 +36,13 @@ NetworkStatisticsWidget::NetworkStatisticsWidget()
         m_adapter_table_view = adapters_group_box.add<GUI::TableView>();
 
         Vector<GUI::JsonArrayModel::FieldSpec> net_adapters_fields;
+        net_adapters_fields.empend("", Gfx::TextAlignment::CenterLeft,
+            [this](const JsonObject& object) -> GUI::Variant {
```

Comment:
Please use east const.
```suggestion
            [this](JsonObject const& object) -> GUI::Variant {
```